src/commcare_cloud/ansible/service_playbooks/kafka.yml

```diff
@@ -8,6 +8,42 @@
     - start_kafka
     - stop_kafka
 
+- name: Delete all Kafka topics
+  become: true
+  hosts: kafka
+  tasks:
+    - name: Set wipe_everything True
+      vars:
+        wipe_everything: 1
+
+    - name: Allow Kafka server to delete topics
+      template: src=server.properties.j2 dest="{{kafka_conf_dir}}/server.properties"
+
+    - name: Get Kafka topics
+      become: yes
+      become_user: "{{ cchq_user }}"
+      django_manage:
+        # TODO: Requires corehq/apps/change_feed/management/commands/list_kafka_topics.py
+        command: 'list_kafka_topics'
+        app_path: "{{ code_home }}"
+        virtualenv: "{{ py3_virtualenv_home }}"
+      register: topics_result
+
+    - name: Delete all the topics
+      loop: "{{ topics_result.stdout_lines }}"
+      # This runs on all Kafka hosts, and assumes Zookeeper is accessible at localhost:2181.
+      # TODO: Is that a valid assumption? (docs/services/kafka.md also makes this assumption.)
```

Comment:
I see this line, which interprets the zookeper location based on the environment config:
https://github.com/dimagi/commcare-cloud/blob/2792cda63f725af47bf9618984422ae7e73511eb/src/commcare_cloud/ansible/roles/datadog/templates/kafka_consumer.yaml.j2#L9-L9